community/inventory_service/item-schema-donations.json

```diff
@@ -967,11 +1003,14 @@
 		{
 			"itemdefid": 100000022,
 			"type": "item",
-			"name": "Donor: ",
```

Comment:
Not having _english is a signal that the string isn't ready to translate yet, but I guess it'd be better to do these with placeholder names instead.